rclpy/rclpy/node.py

```diff
@@ -565,15 +534,14 @@ def declare_parameters(
                 )
 
             if len(parameter_tuple) == 1:
-                warnings.warn(
+                raise TypeError(
```

Comment:
now we can enforce this to raise the exception.

Reply:
I think we can remove `Tuple[str],` from type annotations too

Reply:
Good catch! thanks.